nextcord/file.py

```diff
@@ -13,6 +13,12 @@ class File:
     r"""A parameter object used for :meth:`abc.Messageable.send`
     for sending file objects.
 
+    .. versionadded:: 2.4
```

Comment:
2.5 is the upcoming version now, 2.4 is released.
```suggestion
    .. versionadded:: 2.5
```